src/rsz/test/repair_setup2_multi.ok

```diff
@@ -14,15 +14,15 @@ Iteration |   Area    | Resized | Buffers | Nets repaired | Remaining
 [INFO RSZ-0037] Found 3 long wires.
 [INFO RSZ-0039] Resized 2 instances.
 [INFO RSZ-0038] Inserted 3 buffers in 3 nets.
-[INFO RSZ-0100] Repair move sequence: UnbufferMove SizeMove SwapPinsMove BufferMove CloneMove SplitLoadMove 
+[INFO RSZ-0100] Repair move sequence: UnbufferMove SizeUpMove SwapPinsMove BufferMove CloneMove SplitLoadMove 
 [INFO RSZ-0094] Found 2 endpoints with setup violations.
 [INFO RSZ-0099] Repairing 2 out of 2 (100.00%) violating endpoints...
    Iter   | Removed | Resized | Inserted | Cloned |  Pin  |   Area   |    WNS   |   TNS      |  Viol  | Worst
           | Buffers |  Gates  | Buffers  |  Gates | Swaps |          |          |            | Endpts | Endpt
 --------------------------------------------------------------------------------------------------------------
         0 |       0 |       0 |        0 |      0 |     0 |    +0.0% |   -0.118 |       -0.2 |      2 | y2
         5 |       0 |       0 |        0 |      0 |     0 |    +0.0% |   -0.118 |       -0.2 |      2 | y2
-       10 |       0 |       3 |        0 |      0 |     1 |   +53.7% |   -0.118 |       -0.2 |      2 | y2
+       10 |       0 |       0 |        0 |      0 |     0 |   +53.7% |   -0.118 |       -0.2 |      2 | y2
```

Comment:
This seems suspicious, no? 0 changes but +53% area?

Reply:
Earlier implementations would present data that could be reversed later. I agree it should be consistent though. This isn't limited to my PR but can happen in general.

Reply:
FYI
https://github.com/The-OpenROAD-Project/OpenROAD/pull/7533